City-of-Helsinki/drupal-helfi-kymp
# UHF-X Top bar

To test:

1. Checkout the branch on this repo and also checkout the hdbt repo to a branch that has the same name.
2. `make new`
3. Login to the site and go to add some links to header top navigation: https://helfi-kymp.docker.sh/fi/admin/structure/menu/manage/header-top-navigation/add. Add one internal and one external so that you can see that both work ok.
4. Check that the top bar looks the way it should in the designs.
5. Additional stuff: Try out the site like for example try to add unit search to landing pages etc since a lot of configuration has been updated.

Check out these also:
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/9
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/7
